tools/c7n_org/c7n_org/cli.py

```diff
@@ -438,16 +452,17 @@ def run_account(account, region, policies_config, output_path,
 @click.option('-t', '--tags', multiple=True, default=None)
 @click.option('-r', '--region', default=None, multiple=True)
 @click.option('-p', '--policy', multiple=True)
+@click.option('-l', '--policytags', multiple=True, default=None)
```

Comment:
i wish i could think of a better long form name for this, probably worth a help string on what its doing.

Reply:
Added a help string for account and policy related tags..  Naming things can be hard :)